python/dgl/data/utils.py

```diff
@@ -24,6 +25,22 @@ def _get_dgl_url(file_url):
     return repo_url + file_url
 
 
+def split_dataset(dataset, frac_list=[0.8, 0.1, 0.1], shuffle=False, random_state=42):
```

Comment:
Also maybe you use `42` because this is a magic number, but this can be confusing for people who do not know the story.

Reply:
Changed to None.